notebooks/MEMA_episode_visualisation.ipynb

```diff
@@ -0,0 +1,764 @@
+{
```

Comment:
<p>I would drop this section for now.</p><p> </p><p>Also, you shouldn't use:</p><pre class="ql-syntax" spellcheck="false">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; dat[1]=[dat[1][i]*1/(0.0005) for i in range(len(dat[1]))]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; dat[2]=[dat[2][i]*1/(0.0005) for i in range(len(dat[2]))]
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>